### PR TITLE
fix(rtdb): enable support for firebasedatabase.app

### PR DESCRIPTION
**Pull request by thameezb**
Closes https://github.com/firebase/firebase-admin-go/issues/415

Enables support for Firebase DBs hosted in the EU

### CODE REVIEW

**Comment by thameezb** [comment]
> @googlebot I signed it!

File: db/db.go [hunk 1]
```diff
@@ -48,8 +48,8 @@ func NewClient(ctx context.Context, c *internal.DatabaseConfig) (*Client, error)
 		return nil, err
 	} else if p.Scheme != "https" {
 		return nil, fmt.Errorf("invalid database URL: %q; want scheme: %q", c.URL, "https")
-	} else if !strings.HasSuffix(p.Host, ".firebaseio.com") {
-		return nil, fmt.Errorf("invalid database URL: %q; want host: %q", c.URL, "firebaseio.com")
+	} else if !(strings.HasSuffix(p.Host, ".firebaseio.com") || strings.HasSuffix(p.Host, ".firebasedatabase.app")) {
```
Comment (hiranya911) [inline db/db.go:51]: I think going forward we want to remove this domain validation entirely. That's what we did in our Python SDK. Can we change this PR to just drop the domain validation?

**Review by hiranya911 — CHANGES_REQUESTED** [review]
Sorry for the delayed response. This somehow fell through the cracks. I just had one suggestion. And please rebase agains the dev branch. We don't directly push to the default branch.

File: db/db.go (hunk 1 above)
Comment (thameezb) [reply db/db.go:51]: Done

**Review by thameezb — COMMENTED** [review]
Rebased onto dev

**Review by hiranya911 — APPROVED** [review]
Thanks. LGTM 👍

**Pull request merged**